### PR TITLE
[Feature] 145 README.md update

### LINKED ISSUE #145: [Feature] README.md update

### PR DESCRIPTION
**Pull request by sustainable-git**
# Issue Number
🔒 Close #145 

## Changes

- Changed files
    - README.md

## New features

- New README

## Review points

- Please check out your profiles
- It is different from the template given by the academy, so please compare it.
- And this one is not a final version. I will post images of our app soon.

## References

- [HARO](https://github.com/DeveloperAcademy-POSTECH/Haro)

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,136 @@
-# MC2-Team2-GreedySquirrel
\ No newline at end of file
+# 🏕 Campare
+
+<div align="center">
+
+📆 2022. 05. 10. ~ 2022. 06. 20.
```
Comment (rriver2) [inline README.md:5]: I think we can add our app logo in here !
I export three kinds of logo in sketch and you can choose one from here ~ !
campare logo
campare logo
campare logo

**Review by rriver2 — APPROVED** [review]
thank you smile !

File: README.md [hunk 2]
```diff
@@ -1 +1,136 @@
...
+<br>
+
+## 🎯 Project Goal
+
+1️⃣ Create a great user experience
+
+2️⃣ Improving the car camping experience that has increased to the new normal after Covid hits in Korea
+
+3️⃣ Reducing time exploring, searching, sorting camping equipments before starting camping 
+
```
Comment (rriver2) [inline README.md:29]: create a great user experience through campare app.

-  Improving the car camping experience that has increased to the new normal after Covid hits in Korea

- Reducing time exploring, searching, sorting camping equipments before starting camping 


I think edit like this it better to know our challenge is focus in a great user experience .

**Review by Gobans — APPROVED** [review]
Cool! Thank you smile~! I'd like it

File: README.md (hunk 1 above)
Comment (sustainable-git) [reply README.md:5]: thx :)

Comment (sustainable-git) [reply README.md]: @rriver2 The logo is applied differently when the user's computer is in dark mode and in light mode.

**Review by mijinc — APPROVED** [review]
Thank you for adding more details!

**Review by vveryberry — APPROVED** [review]
job good

**Review by jinwp — APPROVED** [review]
Great job!

**Pull request merged**